pinot-plugins/pinot-input-format/pinot-protobuf/src/main/java/com/google/protobuf/ProtobufInternalUtils.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.google.protobuf;
+
+public class ProtobufInternalUtils {
+  private ProtobufInternalUtils() {
+  }
+  /** convert underscore name to camel name. */
```

Comment:
Explain why we need such conversion instead of just repeating the method name.

Reply:
Linked it in the summary.

Reply:
done